a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -417,7 +414,8 @@ <h4>Variables setup</h4>
                   	<span><b>IF</b> <var>conformance</var> is <b>NOT</b> empty:</span>
                        <span>
                        	<b>THEN LET</b> <var>conformance_string</var> = replace(<var>conformance</var>, ' - ', ' '),
-                       	and <b>LET</b> <var>wcag_level</var> = replace(<var>conformance</var>, 'EPUB Accessibility 1\.1 - WCAG 2\.[0-2] Level ', '').
+                           and <br/>
+                        <b>LET</b> <var>wcag_level</var> = replace(<var>conformance</var>, 'EPUB Accessibility 1\.1 - WCAG 2\.[0-2] Level ', '').
```

Comment:
Won't br halt reading in AT? I think we should be able to do this with a span and some css. It should also be applied to the other multi-variable setting steps in this section. I'll push a change for this when I get a chance.

Reply:
This is fixed in the last commit I made. I just made two separate spans and tweaked the css so that any spans after the first in these variable lists get indented, not just the second one.